README.md

```diff
@@ -18,21 +18,23 @@ And run!
 ```python
 from torch.utils.data import DataLoader
 
-from clloader import CLLoader
+from clloader import ClassIncremental, Permutation, Rotations
```

Comment:
Don't import in the example stuff that you don't use.

Reply:
C'était pour montrer ce qui était possible d'import, mais oui ok

Reply:
I've fixed.

We could write more about those in the documentation.